### ISSUE TITLE
When using the builtin screen reader on Chrome oS VSCode does not read deeply indented code when tabs are used for indentation.

### ISSUE THREAD

**Issue by jareds** [microsoft/vscode#177949]
When using the built in screen reader on Chrome oS with the web version of VSCode lines indented by more then one tab are not read.
Steps to reproduce
1. INsure you are using the CHromevox screen reader on Chrome OS.
2.  Open a sample repository that demonstrates this issue in the web version of VSCode on a Chromebook by visiting https://vscode.dev/github/jareds/vscode-web-chromeos-screenreader-issue
3. Open the spaces file.
4. Read line by line, all lines should be announced.
5. OPen the tabs file.
6. Read line by line, only level 0 and level one lines are announced. ANything with an indentation of two tabs or greater is not announced.

Version: 1.76.2
Commit: ee2b180d582a7f601fa6ecfdad8d9fd269ab1884
User Agent: Mozilla/5.0 (X11; CrOS x86_64 14541.0.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36
Embedder: vscode.dev

**Comment by isidorn** [comment]
Sounds like an issue with the Chromevox screen reader. @jareds can you open an issue for that screen reader please?

**Comment by jareds** [comment]
This appears to be a ChromeOS and Chromevox issue that effects textaria attributes. This issue can probably be closed since it's not VSCode specific. I was able to reproduce the behavior with the link @isidorn  posted. I created a sample and opened the following bug with ChromeOS. https://issuetracker.google.com/issues/274763131

**Comment by isidorn** [comment]
@jareds thank you very much for doing that 👏 
Thus closing this issue
